onboard/catkin_ws/src/controls/launch/position_pid.launch

```diff
@@ -9,17 +9,17 @@
     <!-- ROBOT PID Constants -->
     <group unless="$(arg sim)">
 	   <group ns="controls/x_pos/controller">
-            <param name="Kp" value="1" />
+            <param name="Kp" value=".1" />
```

Comment:
I would assume PID values from competition are better than these old values, so we probably should not merge these

Reply:
Don't merge